tests/ipsec/ipsec3hosts.pm

```diff
@@ -1,138 +1,228 @@
 # SUSE's openQA tests
 #
-# Copyright 2017-2023 SUSE LLC
+# Copyright 2023 SUSE LLC
 # SPDX-License-Identifier: FSFAP
-
-# Summary:  Basic preparation before any IPSec test
+#
+# Summary: Basic preparation before any IPSec test
+# Network topology used in this test:
+#
+#   host(left_ip) ---- middle_ip_01 [ROUTER] middle_ip_02 ---- host(right_ip)
+#
+#   LEFT HOST  <----->  ROUTER  <----->  RIGHT HOST
+#   2001:1:1:1::2       2001:1:1:1::1       2002:1:1:1::2
+#                       2002:1:1:1::1
+#
```

Comment:
nit comments ,maybe format as following is more readable

<img width="782" height="164" alt="image" src="https://github.com/user-attachments/assets/321e22b5-efe7-4583-af88-3a0e88581506" />